src/distance.jl

```diff
@@ -5,6 +5,11 @@ struct DefaultDistance<:AbstractMatrix{Int}
     DefaultDistance(nv::Int=typemax(Int)) = new(nv)
 end
 
+DefaultDistance(nv::Integer) = DefaultDistance(Int(nv))
```

Comment:
The issue is that `nv` now returns `<: Integer`, and I've made `DefaultDistance` just use Ints for simplicity. This converts `nv` to `Int` regardless of the underlying graphtype. It should have no ill effects.